package.json

```diff
@@ -10,8 +10,8 @@
   "scripts": {
     "test": "eslint . && ava -v",
     "lint": "eslint . ",
-    "prepare": "npm run build",
-    "build": "rm -rf ./dist && tsc"
+    "_prepare": "npm run build",
+    "_build": "rm -rf ./dist && tsc"
```

Comment:
These need to be reverted

Reply:
thought we already did, sorry should've caught that myself